graccbackup@.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=GRACC Periodic Backup
+Documentation=https://opensciencegrid.github.io/gracc
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+User=gracc
+Group=gracc
+Type=oneshot
+Environment=IN=%i
+PermissionsStartOnly=true
+ExecStartPre=/bin/bash -c "systemctl set-environment DEST=`grep dest /etc/graccarchive/config/gracc-archive-${IN}.toml | grep -v \\# | cut -d\\' -f2`"
```

Comment:
I don't think we need this anymore.

Reply:
FYI Haven't touched the service file yet, this is still the one generated by puppet in use on the gracc server. Was going to do the backup service and dockerfile changes when I'm ready to test it out on the gracc server.